examples/mongodbatlas_project/main.tf

```diff
@@ -11,4 +11,15 @@ resource "mongodbatlas_project" "test" {
     name  = "atlas.project.deployment.nodesPerPrivateLinkRegion"
     value = 3
   }
+  tags = {
+    Owner       = "Terraform"
+    Environment = "Example"
+    Team        = "tf-experts"
+  }
+
+  lifecycle {
+    ignore_changes = [
+      tags["CostCenter"] # useful if `CostCenter` is managed outside terraform
```

Comment:
CostCenter is not in the tags above, is it?

Reply:
Tried to clarify in 625707e97aac902e62fbc98c34f2bb15c3fb6957
WDYT?